### DISCUSSION TITLE
Razzle deployment to Azure help

### DISCUSSION THREAD

**Discussion by Kaos1105** [jaredpalmer/razzle#1488] [General] [answered]
I am facing the similar issue like this stackoverflow but none have the answer. Can anyone help me out. Much appreciate
https://stackoverflow.com/questions/57134079/what-folders-need-to-be-uploaded-to-deploy-a-razzle-app-on-azure

**Comment by fivethreeo** [comment]
https://stackoverflow.com/questions/54472143/how-to-deploy-a-ssr-react-app-in-azure-app-services

https://gist.github.com/fivethreeo/a1717cef2363783d73b6dd018b621a11

https://gist.github.com/fivethreeo/c294b3837f69d9e77f3189a327ba81b8

Upload build and package.json

**Comment by fivethreeo** [comment, marked as answer]
https://razzle-git-canary.jared.vercel.app/deployment-options/azure

**Reply by Kaos1105** [reply]
thanks a lot mate, i managed to edit the server.js build file. Replace it with PORT.env.process. Then in the build folder, i copy and paste package.json into it, replace the start line with 'node server' then using azure extention to deploy. Work fine but can not CI/CD wth Azure Devops, i have to edit the build manually

**Comment by fivethreeo** [comment]
When 4.0 is out the docs I showed works

**Reply by Kaos1105** [reply]
glad that Razzle still being supported and update frequently. I have struggled nearly a week finding a way to deploy to Azure. Again thanks alot mate
